fe/src/components/Users/Users.re

```diff
@@ -90,4 +136,4 @@ let renderContent = users => {
 let make = (~users, ~isUsersLoading, _children) => {
   ...component,
   render: _self => isUsersLoading ? <LoadingMask /> : renderContent(users),
-};
+};
```

Comment:
Nie jestem w 100% pewien, ale chyba w PSR i tym podobnych jest info, aby każdy plik kończył się jedną pustą linią. Trzeba by to określić, bo będziemy się wzajemnie poprawiać w tym zakresie.

Reply:
@kamilkuz 
To moja wtyczka do reasona w visual studio code tak mi robi po zapisie pliku, niewiemczego i niewiem czy to się będzie dało skonfigurować

Reply:
@kamilkuz 
Hm, dziwne. Wtyczka do storma tą spację dodaje :(. No trudno.